pepper-apis/dsm-core/src/main/resources/liquibase/singular/DDP-195_geneticTestingUpdate.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="DDP-195_UPDATE_GENETIC_TESTING_ANSWER_FIELD" author="nick">
+        <sql>
+            UPDATE `dev_dsm_db`.`field_settings` SET `actions` =
+            '[{\"type\":\"conditionalDisplay\",\"condition\":\"OTHER\",\"conditionalFieldSetting\":{\"columnName\":\"KARYOTYPE_ABNORMAL_OTHER_SPECIFY\",\"fieldType\":\"GROUP_GENETIC_TESTING\",\"columnDisplay\":\"Other Chromosomal anomaly (please specify)\",\"displayType\":\"TEXTAREA\",\"details\":{\"size\":100}}}]'
+            WHERE column_name='KARYOTYPE_ABNORMAL';
+
+        </sql>
+    </changeSet>
+</databaseChangeLog>
```

Comment:
I think checkstyle also wants an empty line in xml files, but not 100% sure
Did you run checkstyle?

Reply:
I did on the most recent commit. Specifically `mvn checkstyle:checkstyle`. Is that the right command?

Reply:
I would suggest specifying the module you're testing as well. For example:
```mvn -pl dsm-server -am checkstyle:checkstyle```

Reply:
alright, I ran that one and changed the spacing to better match the other xml files.